arn/arn.go

```diff
@@ -637,11 +637,7 @@ func MapARNToRTypeAndRName(arnStr ResourceARN) (ResourceType, ResourceName) {
 	var sfx string
 	switch {
 	case strings.HasPrefix(arn, "arn:aws:autoscaling:"):
-		erASG, err := re.Compile("arn:aws:autoscaling:[^:]+:[^:]+:(.+)")
-		if err != nil {
-			fmt.Printf("{\"error\": \"%s\"}\n", err.Error())
-			break
-		}
+		erASG := re.MustCompile("arn:aws:autoscaling:[^:]+:[^:]+:(.+)")
```

Comment:
Switching `Compile` to `MustCompile` because grafiti shouldn't have to handle regex errors arising from compilation of a well-defined pattern (there shouldn't be one).